gomint-server/src/main/java/io/gomint/server/command/CommandManager.java

```diff
@@ -129,29 +147,21 @@ public void executeSystem(String line) {
      * world's thread, if the current thread is not the world's thread.
      *
      * @param sender         of the command
-     * @param command        which should be executed
-     * @param outputConsumer the command output consumer
+     * @param command        which should be executed (includes leading {@code /})
+     * @param outputConsumer the command output consumer, no guarantee is given in which thread it is executed
```

Comment:
The command and outputConsumer should be executed in the entities world thread (or in the gomint mainthread for system commands) this should be ensured with a Precondition safeguard

Reply:
The command itself will be executed in entities world thread.